pkgs/top-level/all-packages.nix

```diff
@@ -30323,7 +30323,7 @@ with pkgs;
   };
 
   jabref = callPackage ../applications/office/jabref {
-    jdk = javaPackages.compiler.openjdk18;
+    jdk = jdk19;
```

Comment:
5.9 was targeted at JDK17 - whatever comes next will use JDK19

Reply:
https://github.com/JabRef/jabref/blob/v5.9/build.gradle#L44 It's 19.

Reply:
Right, 5.9 was cut after https://github.com/JabRef/jabref/pull/9421 was merged.